book-src/05-04-websocket-server.md

```diff
@@ -0,0 +1,7 @@
+## WebSocket
+
+Starting with release 0.14.0, a simple implementation of `http.WebSocket`(Server) has been introduced.
```

Comment:
```suggestion
A basic implementation of `http.WebSocket`(Server) has been introduced since Zig 0.14.0. 

This example demonstrates an echo server.
```